Source/DafnyCore/Verifier/Translator.ExpressionTranslator.cs

```diff
@@ -1772,6 +1772,7 @@ public ExpressionTranslator(ExpressionTranslator etran, string modifiesFrame)
         bool pr;
         if (s is BinaryExpr && aggressive) {
           BinaryExpr bin = (BinaryExpr)s;
+          /*
```

Comment:
Is it ok to remove this code?

Reply:
Yes, unless the test suite shows us otherwise.
When you do remove it, please also remove the unused variable `pr` on L1772.